setup.py

```diff
@@ -110,6 +110,8 @@ def parse_arg_remove_string(argv, arg_name_equal):
     "manylinux2014_s390x",
     "manylinux_2_27_x86_64",
     "manylinux_2_27_aarch64",
+    "manylinux_2_28_x86_64",
```

Comment:
2_27 is for Ubuntu 18.04. You can remove those. We don't have pipelines for them anymore. Ubuntu 18.04 is EOL. We no longer provide support for it. Similarly, once your change is done, we can remove all the above tags.